hs_core/models.py

```diff
@@ -4132,12 +4132,10 @@ def create_date_node(date, date_type):
         dataset_licenses_node = etree.SubElement(dataset_node, '{%s}program' % ai, name="AccessIndicators")
         pub_date_str = pub_date.strftime("%Y-%m-%d")
         rights = self.metadata.rights
-        license_node = etree.SubElement(dataset_licenses_node, '{%s}license_ref' % ai, applies_to="vor",
-                                        start_date=pub_date_str)
         if rights.url:
+            license_node = etree.SubElement(dataset_licenses_node, '{%s}license_ref' % ai, applies_to="vor",
+                                            start_date=pub_date_str)
             license_node.text = rights.url
-        else:
-            license_node.text = rights.statement
```

Comment:
Since we are no more using the rights.statement when the url is missing, should the publishing review process check that the user has provided the url?

Reply:
We could check to ensure that a rights.url is included during publication in [submit_resource_for_review](https://github.com/hydroshare/hydroshare/blob/3.0.1/hs_core/hydroshare/resource.py#L1040) to ensure every published resource has a url. But I guess I don't know that's necessary...

Reply:
I was thinking of when Clara checks for metadata requirements like suitable resource title etc. Just to encourage user to provide url for the license for resource publication. Anyway, I am not sure of the importance of this check.

Reply:
Thanks @pkdash I think that's a good idea!
@ClaraCogswell if possible, please recommend that folks include a license URL (we don't require it in the system) when adding a custom license to the resource that they are publishing.